### ISSUE TITLE
Define Articles that should be included in Title I Chapter II

### ISSUE THREAD

**Issue by Staphylococcus** [Staphylococcus/federal-eu-constitution#20]
A place for discussion related to the overall structure of Title I Chapter II.

What is in scope: 
- List of articles that should Title I Chapter II should comprise of
- Key points to be included across the titles

What is not in scope:
- Formulation of article phrasing

Example outcome:

**Comment by Mathew-Iron** [comment]
@Staphylococcus I agree, we should also draw up an outline for all the chapters before writing them.
